src/global_styling/functions/typography.ts

```diff
@@ -13,6 +13,11 @@ import {
 } from '../variables/typography';
 import { UseEuiTheme } from '../../services/theme/hooks';
 
+export interface _FontScaleOptions {
+  measurement?: _EuiThemeFontSizeMeasurement;
+  customScale?: _EuiThemeFontScale;
```

Comment:
Can we add some code comments especially as to when `customScale` is necesary

Reply:
https://github.com/elastic/eui/pull/5895/commits/ce5052857bff5ec69d66094a07c94fa0cc230de4

Reply:
Heh, I actually meant more like:

```suggestion
  /*
   * ... comment ...
   */
  measurement?: _EuiThemeFontSizeMeasurement;
  customScale?: _EuiThemeFontScale;
```

Reply:
Hmm, I'm not totally clear on your suggestion? Does one of your previous suggestions (https://github.com/elastic/eui/pull/5895#discussion_r877459002) already cover this?

https://github.com/elastic/eui/blob/cfc147198af4be3e7c42111970b70e5c6e006ca1/src/global_styling/functions/typography.ts#L16-L24

Reply:
I mean that these are no longer `@param`s, they're prop comments. Just like in our TS types like:

https://github.com/elastic/eui/blob/09791d7bbb930c856ae870183a7989124b2a5036/src/components/text/text.tsx#L28-L40

So this one would be:

```tsx
export interface _FontScaleOptions {
  /**
   * What string measurement to return.
   */
  measurement?: _EuiThemeFontSizeMeasurement;
  /**
   * An additional custom scale multiplier to use against the current scale.
   * This parameter can be used (e.g. by EuiText sizes) to get sizes of text smaller than the default.
   */
  customScale?: _EuiThemeFontScale;
}
```

Reply:
Ohhh gotcha! I mean technically they are still params (in how they're being used) 🤷 but I'm not super opinionated either way about how we format our comment docs. Will make this change!

Reply:
https://github.com/elastic/eui/pull/5895/commits/a1d2c18c7f1b5329a5155b27e274305982955987